src/panels/config/tags/dialog-tag-detail.ts

```diff
@@ -11,6 +11,7 @@ import { HassDialog } from "../../../dialogs/make-dialog-manager";
 import { haStyleDialog } from "../../../resources/styles";
 import { HomeAssistant } from "../../../types";
 import { TagDetailDialogParams } from "./show-dialog-tag-detail";
+import "../../../components/ha-alert";
```

Comment:
```suggestion
import "../../../components/ha-alert";
import "../../../components/ha-textfield";
```

Paper input above needs removed